mypy/checkmember.py

```diff
@@ -161,6 +162,7 @@ def analyze_member_access(
     module_symbol_table: SymbolTable | None = None,
     no_deferral: bool = False,
     is_self: bool = False,
+    options: Options,
```

Comment:
Since `MessageBuilder` now has `options`, can we use instead of passing `options` around explicitly? We could perhaps use `mx.msg.options` instead, and most of the changes in this file could possibly be reverted.

Reply:
nice, that's much better!